x-pack/packages/kbn-elastic-assistant-common/impl/llm/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { ActionsClientChatOpenAI } from './openai';
+export { ActionsClientLlm } from './actions_client_llm';
```

Comment:
@stephmilovic I use barrel file here instead of index.ts in a parent directory, because elastic-assistant-common maybe called from FE side to that causes an error, because it tried to import opeanAI libraries to fe side